src/analysis/retail/priest/holy/modules/spells/CircleOfHealing.tsx

```diff
@@ -82,6 +82,9 @@ class CircleOfHealing extends Analyzer {
   }
 
   get guideSubsection(): JSX.Element {
+    if (!this.selectedCombatant.hasTalent(TALENTS.CIRCLE_OF_HEALING_TALENT)) {
```

Comment:
So this is a way of doing it but I don't think its the correct way, you should just prevent calling this function by adding a check here https://github.com/WoWAnalyzer/WoWAnalyzer/blob/the-war-within/src/analysis/retail/priest/holy/Guide.tsx#L25

Can follow example from https://github.com/WoWAnalyzer/WoWAnalyzer/blob/87fee3fc4ce93725cb89e8aefdfddc79b247c8bd/src/analysis/retail/monk/mistweaver/Guide.tsx#L29